sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/Embedding.java

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.models;
+
+import com.azure.cosmos.implementation.Constants;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Embedding settings within {@link VectorEmbeddingPolicy}
+ */
+public final class Embedding {
+    @JsonProperty(Constants.Properties.PATH)
+    private String path;
+    @JsonProperty(Constants.Properties.VECTOR_DATA_TYPE)
+    private String vectorDataType;
+    @JsonProperty(Constants.Properties.VECTOR_DIMENSIONS)
+    private Long dimensions;
+    @JsonProperty(Constants.Properties.DISTANCE_FUNCTION)
+    private String distanceFunction;
+
+    /**
+     * Gets the path for the embedding.
+     *
+     * @return path
+     */
+    public String getPath() {
```

Comment:
Do we have a use case to make the getters as public APIs? Usually we start with only setters as public and only make getters public if customers really need them.

Reply:
Retaining the getters as they are essential for consistency. Therefore, even if getPath is not currently utilized, we should keep it here, as per Fabian's suggestion.